solr/core/src/java/org/apache/solr/util/SolrCLI.java

```diff
@@ -384,8 +384,6 @@ else if ("mkroot".equals(toolType))
       return new ZkMkrootTool();
     else if ("assert".equals(toolType))
       return new AssertTool();
-    else if ("utils".equals(toolType))
-      return new UtilsTool();
```

Comment:
Via `git grep -n "[^\.\/a-z]utils[^\.\/a-z]"` found reference in `bin/solr.cmd` file.

Reply:
Thanks, I found a `:run_utils` function in solr.cmd that I removed.